File: C_Programming/Practice/Program7.c
#include<stdio.h>

float AddTwoNumbers(
                        float fNo1,         // First input
                        float fNo2          // Second input
                    )
{
    float fAns = 0.0f;                      // Variable to Store Result 
    fAns = fNo1 + fNo2;                     // Perform Addition
    return fAns;
}

int main()
{
    float fValue1 = 0.0f;                   // To Store First Input
    float fValue2 = 0.0f;                   // To Store Second Input
    float fResult = 0.0f;                   // To Store the Result

    printf("Enter First Number  : ");
    scanf("%f",&fValue1);

    printf("Enter Second Number : ");
    scanf("%f",&fValue1);

    fResult = AddTwoNumbers(fValue1, fValue2);  
    
    printf("Addition is         : %f\n",fResult);

    return 0;
}

File: Java_Programming/Practice/program7.java
import java.util.Scanner;

// Class containing arithmetic methods.
class Arithmatic
{
    public static int AddTwoNumber
                                (
                                    int iNo1,       //First Input 
                                    int iNo2        // Second Input 
                                )
    {
        int iResult = 0;                            // Variable to store Result

        iResult = iNo1 + iNo2;                      // Perform the Addition
    
        return iResult;
    }
}

// Main class of the application.
class program7
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        // Variable Creation with default values
        int iValue1 = 0;   // To Store First Input
        int iValue2 = 0;   // To Store Second Input
        int iResult = 0;   // To Store The Result

        System.out.println("Enter First Number : ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter Second Number : ");
        iValue2 = sobj.nextInt();

        iResult = Arithmatic.AddTwoNumber(iValue1,iValue2);

        System.out.println("Addition is : " + iResult);

        sobj.close();

    }
}